doc/ideas/fr_technical_overview.md

```diff
@@ -33,49 +35,81 @@ Gameplay           | GM8        | villagers need sleep at night, die at work (Ne
 Gameplay           | GM17       | Attack/defense bonuses for some unit formations. Dependent on minimum amount of units in this formation. New marching formation with speed bonus but defense malus. Cavalry flanking.
-Gameplay           | GM18       | Researches only affect newly produced units. Old ones need to be manually upgraded/retrained at production building. Queue for researchings in buildings.
+Gameplay           | GM18       | Researches only affect newly produced units. Old ones need to be manually upgraded/retrained at production building.
 Gameplay           | GM19       | Scale wonder cost to max pop, number of players and available resources.
 Gameplay           | GM20       | a new 5th "free trade" age where allies share resources and give each other massive battles.
 Gameplay           | GM21       | New unit: Lance Cavalry, run over units in defending formations
-Gameplay           | GM22       | Researches can be globally and locally applied. e.g. upgrade just units garrisoned in the barracks for less research cost; e.g. you need 3 pikemen for the battle now, but you don't want to produce them in the future in masses
+Gameplay           | GM22       | Researches can be globally and locally applied. e.g. upgrade just units garrisoned in the barracks for less research cost; e.g. you need 3 pikemen for the battle now, but you don't want to produce them in the future in masses (see GM18)
 Gameplay           | GM23       | Units can level up by gaining experience doing what they normally do, fighting in battle against their conter unit will give more XP against that unit style. Villagers chopping a forest nothing map might be a real pro at chopping trees in the end of the game.
-Unit AI            | UAI1       | Improved grouping. manually coloring units for teammates. designated unit pings its position periodically. double click on a grouped unit in group formation selects same group not all units of same kind. allow more than 10 groups. more intelligent assigning of units to existing groups. sub-group selections.
-Unit AI            | UAI2       | More intelligent units. Emergency evacuation points. Formations and movement when attacking. Dynamic hardlocking on target units when attacking. Aggressive stance for monks (auto-convert). Auto-flee stance. Non-coward stance for villagers. Allow selecting the default stance in production building. Set rally point on moving units.
+Gameplay           | GM24       | Queue for technology researches.
+Gameplay           | GM25       | Farms cannot be placed on desert ground.
+Gameplay           | GM26       | Improved water battles.
+Gameplay           | GM27       | Percentage based armor system similar to AoEO and AoM. Will need to rebalance whole gameplay.
+Gameplay           | GM28       | Reduce the amount of hidden bonuses on units and show them on tooltips. We should easily be able to tell what the bonuses are.
```

Comment:
> Reduce the amount of hidden bonuses on units

This is probably too vague. What's the reasoning behind this?

Reply:
`Reduce the amount of hidden bonuses on units and show them on tooltips. Currently there are so many many hidden bonuses for units it's ridiculous. No unit needs 6 hidden bonuses, talking to you Halbs. It isn't necessary to balance the game even if it makes it slightly easier to do so. And we should easily be able to tell what the bonuses are.`

From reddit, was unclear how to make write more from it. :-P

Reply:
Hmm looks like someone just stated personal preference :D I suggest we remove the "Reduce the amount of hidden bonuses on units" part of the sentence and move the rest of the suggestion to UI.

Reply:
done